config/7.10.0/common.yaml

```diff
@@ -367,23 +367,6 @@ console:
           service: "[[.ApplicationName]]-[[.Console.Name]]"
         annotations:
           description: All the Business Central web server's ports.
-    - spec:
```

Comment:
I see that you didn`t remove it from templates, is there a reason to delete it only here?
Plus, can it be safely removed?

Reply:
I've forgot two templates where was present, are removed because we are removing Openshift DNS jgroups configuration, here the Datagrid and the AMQ configuration
https://access.redhat.com/documentation/en-us/red_hat_data_grid/7.3/html/red_hat_data_grid_for_openshift/os_configuring_clusters

https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.2/html/getting_started_with_jboss_eap_for_openshift_online/reference_information  

do we want preserve the Datagrid and AMQ current config even is based on a deprecated behaviour (Openshift.DNS ping) ?

Reply:
if datagrid and amq docs says to keep using openshift, let's continue to using it. We do not control the amq and datagrid images on this perspective.

Reply:
ok